### PR TITLE
Null check on Flutter View open (#1028).

### PR DESCRIPTION
**Pull request by pq**
See https://github.com/flutter/flutter-intellij/issues/1028.

@devoncarew

### CODE REVIEW

File: src/io/flutter/view/OpenFlutterViewAction.java [hunk 1]
```diff
@@ -39,7 +40,11 @@ public void actionPerformed(@NotNull final AnActionEvent e) {
 
     FlutterInitializer.sendAnalyticsAction(this);
 
-    final ToolWindow toolWindow = ToolWindowManager.getInstance(project).getToolWindow("Flutter");
-    toolWindow.show(null);
+    final ToolWindow toolWindow = ToolWindowManager.getInstance(project).getToolWindow(FlutterView.TOOL_WINDOW_ID);
```
Comment (devoncarew) [inline src/io/flutter/view/OpenFlutterViewAction.java]: Looks like getToolWindow() can return null if `FlutterViewCondition` returns false, which will happen if we don't recognize any modules as flutter modules.

File: src/io/flutter/view/OpenFlutterViewAction.java [hunk 2]
```diff
@@ -39,7 +40,11 @@ public void actionPerformed(@NotNull final AnActionEvent e) {
...
+    if (toolWindow == null) {
+      FlutterMessages.showError("Error Opening Flutter View", "Unable to find Flutter tool window");
```
Comment (devoncarew) [inline src/io/flutter/view/OpenFlutterViewAction.java]: Perhaps modify the message in light of the above? `Unable to open View` and `Unable to open the Flutter tool window - no Flutter modules found.`

**Comment by devoncarew** [comment]
Thanks for the null check! lgtm

File: src/io/flutter/view/OpenFlutterViewAction.java (hunk 2 above)
Comment (pq) [inline src/io/flutter/view/OpenFlutterViewAction.java]: Sure thing.

Related, it seems that maybe we want the `OpenFlutterViewAction` to detect and disable itself in that case?  I may take a little poke at that.

File: src/io/flutter/view/OpenFlutterViewAction.java (hunk 1 above)
Comment (pq) [inline src/io/flutter/view/OpenFlutterViewAction.java]: Got it.  Thanks!

**Pull request merged**